### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by DimaSmagulov**
Home page + css + bootstrap

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,12 @@
...
-end
+
+  # Returns the full title on a per-page basis.
+  def full_title(page_title = '')
+    base_title = "AID-Interview"
+    if page_title.empty?
+      base_title
+    else
+      page_title + " | " + base_title
+    end
```
Comment (Kmalynss) [inline app/helpers/application_helper.rb]: please refactor this method without declaration of ```base_title``` and ```if...else...``` statement

File: Gemfile.lock [hunk 2]
```diff
@@ -288,9 +288,15 @@ GEM
     rb-inotify (0.9.5)
       ffi (>= 0.5.0)
     rdoc (4.2.0)
+<<<<<<< HEAD
```
Comment (freiwillen) [inline Gemfile.lock]: something wrong with this file
I guess there were some problems while rebasing?

hint: you can delete this file and rerun ```bundle``` to generate new one

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -1,2 +1,5 @@
 module ApplicationHelper
-end
+  def full_title(page_title = '')
+  	page_title.empty? ? "AID-Interview" :  page_title + " | " + "AID-Interview"
```
Comment (freiwillen) [inline app/helpers/application_helper.rb]: Can be done in a better way.

hint: 
```ruby
def full_title(page_title = 'nil)
```
then you could use some [array magic](http://docs.ruby-lang.org/en/2.0.0/Array.html) to build a title

**Comment by freiwillen** [comment]
Did you have any problems during rebase? Did you use rebase? Merge commit between yours looks strange. Shouldn't be like this.

File: Gemfile.lock (hunk 2 above)
Comment (DimaSmagulov) [inline Gemfile.lock]: Yes i had a lot of problems with rebase and conflicts)) 
I'll follow your advice

File: app/helpers/application_helper.rb (hunk 3 above)
Comment (DimaSmagulov) [inline app/helpers/application_helper.rb]: ok I'll think about it. But i have no idea how to build it with arrays)

**Comment by DimaSmagulov** [comment]
Now i know hot to do the proper rebase. Next time it'll be ok)

File: public/assets/.sprockets-manifest-bd277d43eaa332dd243c28e926db3b02.json [hunk 4]
```diff
@@ -0,0 +1 @@
...5d3458bc3c8.svg","font-awesome/fonts/fontawesome-webfont.ttf":"font-awesome/fonts/fontawesome-webfont-4d6eb9e9d852a2a6f74e7c428456a2f07fc63a1613d10192d8ed3401d9da5ffa.ttf","font-awesome/fonts/fontawesome-webfont.woff":"font-awesome/fonts/fontawesome-webfont-199411f659f41aaccb959bacb1b0de30e54f244352a48c6f9894e65ae0f8a9a1.woff","web_console/application.js":"web_console/application-37bc1d0490c1c65cb8a777119d058e0b2a29424b5df44db84c308e73828961f3.js","web_console/application.css":"web_console/application-03ccc0db7d3e10231bb3b8b9a780be36587749066b3011b0525b805baeb162c1.css","bootstrap/glyphicons-halflings-regular.eot":"bootstrap/glyphicons-halflings-regular-f495f34e4f177cf0115af995bbbfeb3fcabc88502876e76fc51a4ab439bc8431.eot","bootstrap/glyphicons-halflings-regular.svg":"bootstrap/glyphicons-halflings-regular-d168d50a88c730b4e6830dc0da2a2b51dae4658a77d9619943c27b8ecfc19d1a.svg","bootstrap/glyphicons-halflings-regular.ttf":"bootstrap/glyphicons-halflings-regular-bd18efd3efd70fec8ad09611a20cdbf99440b2c1d40085c29be036f891d65358.ttf","bootstrap/glyphicons-halflings-regular.woff":"bootstrap/glyphicons-halflings-regular-fc969dc1c6ff531abcf368089dcbaf5775133b0626ff56b52301a059fc0f9e1e.woff"}}
```
Comment (serhiym) [inline public/assets/.sprockets-manifest-bd277d43eaa332dd243c28e926db3b02.json]: @freiwillen @DimaSmagulov  do we need push compiled assets?

Comment (DimaSmagulov) [inline public/assets/.sprockets-manifest-bd277d43eaa332dd243c28e926db3b02.json]: sometimes i have problems on heroku with assets

**Pull request merged**